### ISSUE TITLE
will Rainloop be further developed or what does it look like?

### ISSUE THREAD

**Issue by Mululu** [RainLoop/rainloop-webmail#1880]
will Rainloop be further developed or what does it look like?

**Comment by fluffyegg** [comment]
I agree, I'd also like to know this. No updates for a year! We've fixed a few bugs on our own installation that we'd like to roll back into the general release. If this project is dead, we'd love to take over the repo and continue to develop it.

**Comment by plastikschnitzer** [comment]
Normally, Rainloop got multiple updates per year since 2014. Is there any official message from the developers? Do they still sell commercial licenses as stated on the website?

**Comment by FireFtw** [comment]
Also would love to hear something, the lack of communication alone is enough for me to support someone else taking over honestly.

**Comment by jd666** [comment]
1.13.0 was released 22 days ago - so there is development going on.
I agree that there is a lack of communication - however, as I am not a paying customer (I only use it for a single mailbox), I am not complaining about it. I just put a watcher on the github project and get notified when something relevant happens.

**Comment by rabesocke** [comment]
I don't know if the last version is an indicator for ongoing development. There were only merges of two pull requests #1889 and #1888 for this new version. Beside this there was no really change in the repository.

**Comment by czPechy** [comment]
@RainLoop

**Comment by phsc84** [comment]
I strongly recommend [Snappymail](https://github.com/the-djmaze/snappymail), which is actively maintained.

**Comment by Mululu** [comment]
Thank you very much! I will now switch to SnappyMail.
